full-stack-tests/core/src/backend/backend.ts

```diff
@@ -99,15 +97,15 @@ async function init() {
     await server.initialize(port);
     console.log(`Web backend for full-stack-tests listening on port ${port}`);
 
-    await new Promise((resolve) => {
-      http.createServer(async (request, response) => {
-        return serveHandler(request, response, {
-          cleanUrls: false,
-          public: "lib",
-          headers: [{ source: "*", headers: [{ key: "Access-Control-Allow-Origin", value: "*" }] }],
-        });
-      }).listen(Number(process.env.CERTA_PORT ?? 3011) + 4000, undefined, undefined, resolve);
-    });
+    // await new Promise((resolve) => {

```

Comment:
Planning to resurrect commented-out code in future?

Reply:
Removed after latest merge with master.